modules/ROOT/content-nav.adoc

```diff
@@ -16,8 +16,7 @@ Generic Start
 ** xref:managing-instances/migration-readiness.adoc[Migration Readiness Report]
 ** xref:managing-instances/develop.adoc[Develop]
 ** xref:managing-instances/regions.adoc[Regions]
-
-//(tapping on resources will take you to the metrics tab, and then I fully document the metrics tab further down and I link to that in my notes)
+** xref:managing-instances/billing.adoc[Billing]
```

Comment:
I don't think that this belongs here. It should be its own section in the left-hand nav. It is not interactive, so there is nothing to manage.